mozilla-release/mobile/android/app/build.gradle

```diff
@@ -528,3 +529,20 @@ android.applicationVariants.all { variant ->
     def buildType = "${variant.buildType.name.capitalize()}"
     tasks["compile${productFlavor}${buildType}UnitTestSources"].dependsOn(tasks["merge${productFlavor}${buildType}Assets"])
 }
+preBuild {
```

Comment:
Add an empty line here

Reply:
And add also
```java
/* Cliqz start */
...
/* Cliqz end */
```

See [docs](https://github.com/cliqz-oss/cliqz-android/wiki/Coding-style)